src/Propel/Generator/Builder/Sql/Pgsql/PgsqlDataSQLBuilder.php

```diff
@@ -1,99 +0,0 @@
-<?php
-
-/**
- * This file is part of the Propel package.
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @license MIT License
- */
-
-namespace Propel\Generator\Builder\Sql\Pgsql;
-
-use Propel\Generator\Builder\Sql\DataSQLBuilder;
-use Propel\Generator\Builder\Util\DataRow;
-use Propel\Generator\Model\IdMethod;
-
-/**
- * PostgreSQL class for building data dump SQL.
- *
- * @author Hans Lellelid <hans@xmpl.org>
- */
-class PgsqlDataSQLBuilder extends DataSQLBuilder
```

Comment:
Deleted intentionally? Is it possible someone consuming propel is using this?

Reply:
No, since the class this one extends also didn't exist at all. So this was 100% dead code.

Reply:
:+1: excellent